gui/journal.lua

```diff
@@ -164,9 +201,9 @@ function JournalWindow:loadConfig()
 
     local table_of_contents = copyall(journal_config.data.toc or {})
     table_of_contents.width = table_of_contents.width or 20
-    table_of_contents.visible = table_of_contents.visible or false
+    table_of_contents.visible = table_of_contents.visible or true
```

Comment:
this expression is equivalent to `table_of_contents.visible = true`. is that what you want here?